app/utils/chart-config/versioning.ts

```diff
@@ -1790,6 +1795,67 @@ export const configuratorStateMigrations: Migration[] = [
       return newConfig;
     },
   },
+  {
+    description: "ALL (add color field to Chart)",
+    from: "4.0.0",
+    to: "4.1.0",
+    up: (config) => {
+      const newConfig = {
```

Comment:
As discussed today, it'd be better to have a "bump chart config version" migration here, and do the actual migration of chart config separately 👍